src/components/Common/Fields/NumberInputField/NumberInputField.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import type { Story } from '@ladle/react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { NumberInputField } from './NumberInputField'
+import { LocaleProvider } from '@/contexts/LocaleProvider'
+
+interface FormWrapperProps {
+  children: React.ReactNode
+  defaultValues?: {
+    test?: number
+    amount?: number
+    quantity?: number
+    percentage?: number
+  }
+}
+
+const FormWrapper = ({ children, defaultValues = {} }: FormWrapperProps) => {
```

Comment:
Should we maybe put this somewhere modular if we're just using it for Ladle testing?

Reply:
Yeah maybe we can consolidate in one of the follow up tickets for hook form separation? I can look to update it to a single place when doing checkbox group